app/src/main/java/ch/epfl/skysync/navigation/HomeGraph.kt

```diff
@@ -29,12 +29,14 @@ import ch.epfl.skysync.screens.flightDetail.FlightDetailScreen
 import ch.epfl.skysync.viewmodel.ChatViewModel
 import ch.epfl.skysync.viewmodel.FlightsViewModel
 import ch.epfl.skysync.viewmodel.MessageListenerSharedViewModel
+import ch.epfl.skysync.viewmodel.TimerViewModel
 
 /** Graph of the main screens of the app */
 fun NavGraphBuilder.homeGraph(
     repository: Repository,
     navController: NavHostController,
-    uid: String?
+    uid: String?,
+    timer: TimerViewModel? = null
```

Comment:
Why make timer nullable in `homeGraph` when it is not in `MainGraph` and thus always passed ?

Reply:
this was just an easy workaround to not change all the tests

Reply:
Please state it explicitly in a comment then